fe/pom.xml

```diff
@@ -20,29 +20,60 @@ under the License.
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>18</version>
+    </parent>
 
-    <groupId>org.apache</groupId>
-    <artifactId>doris-fe</artifactId>
-    <version>3.4.0</version>
+    <groupId>org.apache.doris</groupId>
+    <artifactId>fe</artifactId>
+    <version>0.15</version>
```

Comment:
It's better to use `0.15-SNAPSHOT`? Since it's more like a standard we treat the SNAPSHOT qualifier as "as-yet-unreleased" version in continuous integration.

Reply:
this is ok, cause this is actually not for user to use.